doc/source/WorkingWithScenes/LightLinking/index.md

```diff
@@ -0,0 +1,176 @@
+
+> Important :
+> The _Linked Lights_ plug is an exclusive list. If you toggle the plug and leave the list blank, the object will not be illuminated by any lights **at all**. 
+> In order to link a particular light **in addition to** all default lights to an object, you must include `defaultLights` in the set expression, as in `defaultLights /group/light`.
+
+
+### Linking by location ###
+
+To link a light by location, simply add the light's full location to the set expression. No additional node is needed.
+
+![A light linking set expression with a location](images/taskLightLinkingSetExpressionLocation.png "A light linking set expression with a location")
+
+> Important :
+> When linking a light by its location, the path you provide **must** be the final location of the light within the hierarchy **as delivered to the render node**. For example, if you link to a light at `/group/light`, but later in the graph you shuffle the hierarchy and the light exists at `/building/lights/light`, the renderer will look for the light at the original location, find nothing there, and then the link will fail. Therefore, you would unintuitively need to target a downstream location in an upstream part of the graph.
+
+
+### Linking by set ###
+
+In setups with more than a few lights, managing a set expression with multiple light locations is prone to user error, and tedious to maintain. That's why it's usually better to link objects to one or more sets containing lights, as the term _set expression_ suggests. Put another way, you can perform **set**-centric light linking. The end result would still be the same, in that specific objects are linked to specific lights.
+
+A light can be added to any number of custom sets using the _Sets_ plug on the light's node, or a separate Set node. Custom sets are separate and do not affect the "defaultLights" set.
+
+![A Set node downstream of a light node in the Graph Editor](images/interfaceLightSetGraphEditor.png "A Set node downstream of a light node in the Graph Editor")
+
+![A Set node in the Node Editor](images/interfaceLightSetNodeEditor.png "A Set node in the Node Editor")
+
+To link one or more lights by a set, simply add it to the set expression.
+
+![A light linking set expression with a set](images/taskLightLinkingSetExpressionSet.png "The light linking set expression with a set")
+
+Be aware that if an object belongs to multiple sets with light linking, the object will only inherit the `linkedLights` attribute from the last set that was linked in the graph.
+
+![A graph where an object belongs to multiple sets, each with different linked lights](images/illustrationLightLinkingMultipleSets.png "A graph where an object belongs to multiple sets, each with different linked lights")
+
+
+## Example Scenario ##
+
+Let's explore an example scenario where you would need to link lights.
```

Comment:
Sounds reasonable. I didn't think the wording was that prescriptive, but I'll defer to your judgement.

Reply:
Thanks, it's just there are so many ways/reasons to do particular things in VFX and techniques evolve rapidly. It's a Gaffer user's job to know what approach to use in each situation. As such, unless we're talking about something specific to Gaffer that directly affect _its_ operation (eg: OSL .vs. Python in expressions), our docs should always be about explaining how something works in Gaffer, and how to do it - rather than why you might want to do it. It's a fine line with examples are you're trying to make them _seem_ production-relevant. Generally though, they always end up being somewhat contrived, so its best to err on the side of caution.  Real shots are always way more complex, so it's easy to sound like we're teaching them how to suck eggs. For example, in this case - if someone doesn't know _why_ they need light linking, I don't think we should try to make Gaffers docs the place they learn - there are lots of other resources for that.